Microsoft.Toolkit.Uwp/IncrementalLoadingCollection/IncrementalLoadingCollection.cs

```diff
@@ -159,6 +160,18 @@ public IncrementalLoadingCollection(int itemsPerPage = 20, Action onStartLoading
         /// <seealso cref="IIncrementalSource{TSource}"/>
         public IncrementalLoadingCollection(TSource source, int itemsPerPage = 20, Action onStartLoading = null, Action onEndLoading = null, Action<Exception> onError = null)
```

Comment:
This ctor shouldn't be changed. The ctor that does not include the TSource should be changed to no call this ctor and to try to create the TSource